justInServer.java

```diff
@@ -1,3 +1,37 @@
+import java.io.IOException;
+import java.net.*;
+import java.sql.*;
+
 public class justInServer {
-//  todo
+
+
 }
+
+class JustInServerSocket   {
```

Comment:
Why is this here? we don't need an entire new class just for a JustinServer, put it in the constructor for the actual JustinServer, and add the socket as an iVar. Also, don't push any code unless its properly commented